docs/conf/eu/2017/news/2week-info.rst

```diff
@@ -0,0 +1,45 @@
+
+- **Sunday September 10th - Writing Day and Welcome Reception**. Both the `Writing Day <http://www.writethedocs.org/conf/eu/2017/writing-day/>`_ and the Welcome Reception will be held at the main conference venue, `Auto Klub, Opletalova 1337/29, Praha 1 <https://goo.gl/maps/dcbYPV5x5DN2>`_. We will also hold a `Welcome Wagon <http://www.writethedocs.org/conf/eu/2017/welcome-wagon/>`_ orientation during the receiption to help ease your way into the conference.
+
+- **Monday September 11th - Conference Day 1**. This is the main event! Come to `Auto Klub, Opletalova 1337/29, Praha 1 <https://goo.gl/maps/dcbYPV5x5DN2>`_ and make yourselves at home. Doors open at 9:00 for breakfast, kick-off announcements at 10:00, day wraps up at 18:00. Post-lunch `lightning talks <http://www.writethedocs.org/conf/eu/2017/lightning-talks/>`_ included, and the `unconference sessions <http://www.writethedocs.org/conf/eu/2017/unconference/>`_ will start in the afternoon.
+
+- **Monday September 11th - Social Event**. We will hang out at the `Hangar Bar, Dušní 9/9, Praha 1 <https://goo.gl/maps/qDfi6bqyn212>`_ from **20:00**. Light snacks and drinks are included while our tab lasts. **Please bring your conference badge**, as entrance is only for conference attendees.
+
+- **Tuesday September 12th - Conference Day 2**. Another great day for documentarians! By now you are already familiar with `Auto Klub <https://goo.gl/maps/dcbYPV5x5DN2>`_ and know your way around. Again, doors open at 9:00 for breakfast, kick-off announcements at 10:00, day wraps up at 17:00. Post-lunch `lightning talks <http://www.writethedocs.org/conf/eu/2017/lightning-talks/>`_ included, and the `unconference sessions <http://www.writethedocs.org/conf/eu/2017/unconference/>`_ will run all day.
+
+For the full schedule overview including start/end times for all activities, take a look at our `Schedule <http://www.writethedocs.org/conf/eu/2017/schedule/>`_ page.
+
+For practical information about Prague, check out the `Visiting Prague <http://www.writethedocs.org/conf/eu/2017/visiting/>`_
+
+And if you still have questions, perhaps you'll find the answers in our `Welcome Wagon <http://www.writethedocs.org/conf/eu/2017/welcome-wagon/>`_ guide. You're also welcome to reach out to @praguewelcomewagon on Slack or send us an email at europe@writethedocs.org.
+
+Badges and Shirts Registration Ends August 31st
+-----------------------------------------------
+
+Even though we still have a few `tickets <http://www.writethedocs.org/conf/eu/2017/tickets>`_ left, we are getting closer to the cut-off time for badges and shirts.
+Please note that if you purchase tickets after **August 31st, 2017**, we will not be able to guarantee a badge or a shirt.
```

Comment:
We should probably mention a badge with your name printed, they will still have one, just with a hand written name.